gridstatus/spp.py

```diff
@@ -193,6 +205,146 @@ def get_load_forecast(self, date, forecast_type="MID_TERM", verbose=False):
 
         return current_day_forecast
 
+    def get_solar_and_wind_forecast_short_term(self, date, end=None, verbose=False):
+        """
+        Returns solar and wind generation forecast for +4 hours in 5 minute intervals.
+        Include actuals for past day in 5 minute intervals.
+
+        Arguments:
+            date (pd.Timestamp|str): date to get data for. Supports "latest" and "today"
+            verbose (bool): print info
+
+        Returns:
+            pd.DataFrame: forecast as dataframe.
+        """
+        # The short_term forecast is delayed up to 2 minutes.
+        buffer_minutes = 2
+
+        if date == "latest":
+            date = self.now() - pd.Timedelta(minutes=buffer_minutes)
+
+        df = self._retrieve_solar_and_wind_forecast_short_term(date, end, verbose)
```

Comment:
not clear to me why we need this separate method. can't we just wrap `get_solar_and_wind_forecast_short_term` with the decorator?

Reply:
Hmm, yeah that should be possible because `support_date_range` just calls the function with "latest" when `date="latest"`.